test/ds/test_dynamic_shape_models.py

```diff
@@ -155,6 +155,35 @@ def create_dynamic_test_data(self,
     y_test_nonzero_dev = torch.nonzero(y_test_xla.int()).float().squeeze()
     return x_test_nonzero_dev, y_test_nonzero_dev
 
+  def test_roialign(self):
+    # For a python implementation of RoIAlign, refer to https://dev-discuss.pytorch.org/t/a-pure-python-implementation-of-roi-align-that-looks-just-like-its-cuda-kernel/1266
+    # specifically https://gist.github.com/ezyang/813e86ff5b46ae9e41fc1920790e51fc.
+    # Create a feature map with size (1, 256, 16, 16).
+    # This could be an output from a convolutional layer of a CNN
+    features = torch.randn(1, 256, 16, 16, device=xla_dev)
+
+    # Create regions of interest
+    # Each RoI is defined by a tuple (idx, x1, y1, x2, y2)
+    # idx is the index into features indicating which image the RoI belongs to
+    # (x1, y1) and (x2, y2) are the coordinates of the top-left and bottom-right corners of the RoI respectively
+    rois = torch.tensor([[0, 60, 60, 100, 100], [0, 120, 120, 160, 160]],
+                        dtype=torch.float,
+                        device=xla_dev)
+
+    # Set output size and spatial scale
+    output_size = (7, 7)
+    spatial_scale = 1.0 / 16.0
+
+    from torchvision.ops import roi_align as roi_align_torchvision
+    print(
+        roi_align_torchvision(features, rois, output_size, spatial_scale).sum())
```

Comment:
why print here?

Reply:
It'll execute the HLO graph and show the result. Later when we check the correctness, we'll compare it with the aten result.